components/Table/TableKDA.css

```diff
@@ -0,0 +1,19 @@
+@import '../palette.css';
```

Comment:
These should probably go in a separate directory.  I think everything in the Table directory should be general-purpose Table code, not Dota-specific things like KDA elements.